pyproject.toml

```diff
@@ -14,9 +14,10 @@ classifiers = [
   "Operating System :: OS Independent",
 ]
 authors = [
-  { name = "Phil Elson", email = "philip.elson@cern.ch" },
+  { name = "Phil Elson", email = "pelson.pub@gmail.com" },
 ]
 requires-python = ">=3.9"
+license = "Apache-2.0"
```

Comment:
Not valid for the latest setuptools which supports legacy Python (Python 3.8). I am going to refactor the workflow to build against the latest Python, and test the wheel with older Pythons.